rocks-bin/src/outdated.rs

```diff
@@ -0,0 +1,76 @@
+use std::collections::HashMap;
+
+use clap::Args;
+use eyre::{OptionExt as _, Result};
+use itertools::Itertools;
+use rocks_lib::{
+    config::Config,
+    manifest::{manifest_from_server, ManifestMetadata},
+    tree::Tree,
+};
+use text_trees::{FormatCharacters, StringTreeNode, TreeFormatting};
+
+#[derive(Args)]
+pub struct Outdated {
+    #[arg(long)]
+    porcelain: bool,
+}
+
+pub async fn outdated(outdated_data: Outdated, config: &Config) -> Result<()> {
+    let tree = Tree::new(
+        &config.tree,
+        config
+            .lua_version
+            .as_ref()
+            .ok_or_eyre("lua version not supplied!")?,
```

Comment:
suggestion: add some help to the error message. Perhaps something like: `"lua version not set. You can supply it via '--lua-version' or set it in the config."`